root/usr/lib/systemd/system/slapd-create-DH2048.service

```
@@ -0,0 +1,7 @@
+[Unit]
+Description=Create the DH2048 group for slapd
+ConditionPathExists=!/etc/openldap/certs/slapd.dh.params
+
+[Service]
+Type=simple
+ExecStart=/usr/libexec/nethserver/slapd-create-DH2048
```

The number of bits may be changed in the future: I'd prefer a "neutral" name

```suggestion
ExecStart=/usr/libexec/nethserver/slapd-create-dhparam
```

---

@DavidePrincipi: I think that @stephdl has based the name with:
- https://github.com/NethServer/nethserver-mail/blob/master/common/usr/lib/systemd/system/postfix-create-DH2048.service
- https://github.com/NethServer/nethserver-mail/blob/master/common/usr/libexec/nethserver/postfix-create-DH2048

---

Note: Currently:
- https://github.com/search?q=org%3ANethServer+DH1024&type=code: nethserver-mail + nethserver-vpn + nethserver-openvpn
- https://github.com/search?q=org%3ANethServer+DH2048&type=code: nethserver-mail + nethserver-openvpn